### PR TITLE
Add unit tests for caffe DB and keras import_json

### PR DESCRIPTION
**Pull request by owenl131**

### CODE REVIEW

**Comment by utsavgarg** [comment]
@owenl131 thanks for the PR, will review soon.

**Comment by owenl131** [comment]
Hi @utsavgarg how is it?

File: tests/unit/caffe_app/test_db.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        tests = open(os.path.join(settings.BASE_DIR, 'tests', 'unit', 'ide',
+                                  'caffe_export_test.json'), 'r')
+        net = json.load(tests)['net']
+        response = self.client.post(
+            reverse('saveDB'),
+            {'net': net, 'net_name': 'netname'})
+        response = json.loads(response.content)
+        self.assertEqual(response['result'], 'success')
+
+    def test_save_jsonstring(self):
```
Comment (utsavgarg) [inline tests/unit/caffe_app/test_db.py]: This is not required.

File: tests/unit/keras_app/test_views.py [hunk 2]
```diff
@@ -35,17 +38,40 @@ def setUp(self):
...
+    def test_keras_import_sample_id(self):
+        response = self.client.post(
+            reverse('keras-import'),
+            {'sample_id': 'vgg16'})
+        response = json.loads(response.content)
+        self.assertEqual(response['result'], 'success')
+        self.assertEqual(response['net_name'], 'vgg16')
+        self.assertTrue('net' in response)
+
+    def test_keras_import_invalid_sample_id(self):
```
Comment (utsavgarg) [inline tests/unit/keras_app/test_views.py]: Either split the Caffe `sample_id` tests into two or combine these into one for uniformity

File: tests/unit/caffe_app/test_db.py (hunk 1 above)
Comment (owenl131) [reply tests/unit/caffe_app/test_db.py]: Which part should be removed?

Comment (utsavgarg) [reply tests/unit/caffe_app/test_db.py]: The `test_save_jsonstring` function as both of them cover the same code.

Comment (owenl131) [reply tests/unit/caffe_app/test_db.py]: One is with the net passed in as a string while the other is passed in as an object. Not sure whether there is any difference in the underlying mechanism.

Comment (owenl131) [reply tests/unit/caffe_app/test_db.py]: Updated :D

**Comment by utsavgarg** [comment]
LGTM :)

**Pull request merged**